#include<iostream>
#include<string>
using namespace std;
int main( void ){
	try{
		try{
			string ex("exception" );
			throw ex;
		}catch( int &ex ){
			cout << "Inside nested catch block" <<endl;
		}
	}catch( string &ex ){
		cout << "Inside outer catch block" <<endl;
	}
	return 0;
}

